nanofi/include/cxx/CallbackProcessor.h

```diff
@@ -65,7 +65,7 @@ class CallbackProcessor : public core::Processor {
 
  public:
 
-  void setCallback(void *obj,std::function<void(processor_session*)> ontrigger_callback) {
+  void setCallback(void *obj,std::function<void(core::ProcessSession*)> ontrigger_callback) {
```

Comment:
It's been a while since I've looked at that work, but did the python bindings call this correctly when running python processors?

Reply:
Yes, tested and works.